app/main.go

```diff
@@ -50,14 +50,17 @@ type HTTPTransportConfig struct {
 	KeepAlive             time.Duration
 }
 
+// SHA256Sums alias for the sha256 sums
+type SHA256Sums []string
```

Comment:
made this because I think I need to `kong.Bind` it too

Reply:
because https://github.com/cashapp/hermit/pull/118/files#diff-54f9129c1a376afcd83728c59ff47e91d9de0b2672d4312179bf28e75846d9d5R18 . and because reflection, this only comes up at runtime. would be good to have some integration test that shells out and actually runs the commands and checks expected results